pkg/apis/eksctl.io/v1alpha5/types.go

```diff
@@ -704,15 +704,15 @@ type (
 	// NodeGroupInstancesDistribution holds the configuration for spot instances
 	NodeGroupInstancesDistribution struct {
 		//+required
-		InstanceTypes []string `json:"instanceTypes,omitEmpty"`
+		InstanceTypes []string `json:"instanceTypes,omitempty"`
```

Comment:
due to this misspelled, these fields are considered as `required` in the schema. I believe that the intention is to have them as optional. Hence, I have updated schema for the same. Let me know if you have other idea.